src/deploy/comet.ts

```diff
@@ -22,14 +28,17 @@ export async function deployComet(txParams: TxParams): Promise<CometContract> {
   const cometFactoryAddress = await deployContract('cometFactory', 'cometFactory', txParams);
   await bumpContractInstance('cometFactory', txParams);
   const cometFactory = new CometFactoryContract(cometFactoryAddress);
-  await invokeSorobanOperation(
-    cometFactory.init(txParams.account.accountId(), comet_wasm),
-    () => undefined,
-    txParams
-  );
+  await invokeSorobanOperation(cometFactory.init(comet_wasm), () => undefined, txParams);
```

Comment:
Comet factory setup should be seperated no?

it feels like one script would be:

1. Deploy comet
-> install both contracts and call init on CometFactory

2. Deploy BLND_USDC Pool
 -> call newCometPool with the fixed params for a BLND-USDC pool (weights and swap fee hardcoded)
 -> intake blndAddress, usdcAddress, and balances as args